pkgs/tools/misc/teleconsole/deps.nix

```diff
@@ -0,0 +1,11 @@
+[
+    {
+      goPackagePath = "github.com/gravitational/teleport";
+      fetch = {
+        type = "git";
+        url = "https://github.com/gravitational/teleport";
+        rev = "2cb40abd8ea8fb2915304ea4888b5b9f3e5bc223"; # v2.0.0-alpha.4
```

Comment:
Can you add a comment that this is asked for here https://github.com/gravitational/teleconsole/blob/09591f227c2a8df4c68af8bc4adfadfc596f4ed2/Makefile#L8 ?

I assume this will be vendored eventually.